sdk/communication/Azure.Communication.PhoneNumbers/src/InternalPhoneNumbersClient.cs

```diff
@@ -1,11 +1,16 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Threading;
 using Azure.Core;
 
 namespace Azure.Communication.PhoneNumbers
 {
     [CodeGenModel("PhoneNumbersClient")]
+    [CodeGenSuppress("GetOperationAsync", typeof(string), typeof(CancellationToken))]
```

Comment:
The sdk does not provide the GetOperation, CancelOperation functionalities so they are suppressed for now.